test/unit_tests/apps/youth-employment-success/js/views/days-per-week-spec.js

```diff
@@ -97,12 +97,14 @@ describe( 'DaysPerWeekView', () => {
   describe( 'on state update', () => {
     it( 'removes u-hidden class when transportation mode is drive', () => {
       const state = {
-        routes: [ {
-          transportation: 'Drive'
-        } ]
+        routes: {
+          routes: [ {
```

Comment:
`routes.routes` is correct?

Reply:
it is! routes is, somewhat confusingly, the name of both the state domain and the values of that domain. I'm going to refactor that in another PR as soon as I get a chance!

Luckily this is at its goofiest in the tests, since all the code uses a selector to access the routes data.